case-study/sqls/working-hour-distribution/sql

```diff
@@ -0,0 +1,12 @@
+SELECT 
+  COUNT(*) AS count, 
+  anyHeavy(repo_name) AS name,
+  anyHeavy(repo_id) as id,
+  toDayOfWeek(created_at) AS dayOfWeek, 
+  toHour(created_at) AS hour 
+FROM 
+  {{table}} 
+WHERE 
+  repo_id IN [{{repoIdList}}] 
```

Comment:
By this SQL, you will select working hour of all the repos, but we may need to generate single working hour image for every repo in the case study.

Reply:
Yes, but in post-processor.js I distribute the result by repo_id and generate a list of svg links which have contained the <embed> tag.

Reply:
OK, my mistake, it looks fine here.